setup.cfg

```diff
@@ -23,6 +23,8 @@ install_requires =
     click>=7.0
     diskcache>=5.2.1
     typing-extensions>=4.1.1
+    pymongo>=3.9.0
+    pydantic[dotenv]>=1.9.0
```

Comment:
Minor comment: I usually try to keep the dependencies in alphabetical order, to make maintenance easier, if you can just change that.

Reply:
sure will do